docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,842 @@
+this.RaisePropertyChanged<MyObject>(() => this.MyProperty, this.oldValue, this.newValue, true);
+
+// Toolkit.Mvvm
+this.OnPropertyChanged<MyObject>(nameof(this.MyProperty));
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+```
+
+#### ICleanup.Cleanup ()
+
+There is no direct replacement for the `ICleanup` interface.
+
+However, the `ObservableRecipient` provides an `OnDeactivated` method which should be used to provide the same functionality as `Cleanup`. 
+
+`OnDeactivated` in the MVVM Toolkit will also unregister all of the registered messenger events when called.
+
+```csharp
+// MvvmLight
+this.Cleanup();
+
+// Toolkit.Mvvm
+this.OnDeactivated();
+```
+
+Note, the `OnActivated` and `OnDeactivated` methods can be called from your existing solution as with `Cleanup`. 
+
+However, the `ObservableRecipient` exposes an `IsActive` property that also controls the call to these methods when it is set. 
+
+### ViewModelBase Properties
+
+#### MessengerInstance
+
+`MessengerInstance` has a renamed direct replacement, `Messenger`.
+
+```csharp
+// MvvmLight
+IMessenger messenger = this.MessengerInstance;
+
+// Toolkit.Mvvm
+IMessenger messenger = this.Messenger;
+```
+
+#### IsInDesignMode
+
+There is no direct replacement for the `IsInDesignMode` property and any code using this should be altered or removed.
+
+The reason for the omission from the MVVM Toolkit is that the `IsInDesignMode` property exposed platform-specific implementations. The MVVM Toolkit has been designed to be platform agnostic.
+
+```csharp
+// MvvmLight
+var isInDesignMode = this.IsInDesignMode;
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+### ViewModelBase Static Properties
+
+#### IsInDesignModeStatic
+
+There is no direct replacement for the `IsInDesignModeStatic` property and any code using this should be altered or removed.
+
+The reason for the omission from the MVVM Toolkit is that the `IsInDesignMode` property exposed platform-specific implementations. The MVVM Toolkit has been designed to be platform agnostic.
+
+```csharp
+// MvvmLight
+var isInDesignMode = ViewModelBase.IsInDesignModeStatic;
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+## Migrating RelayCommand
+
+The following steps focus on migrating your existing components which take advantage of the `RelayCommand` of the MvvmLight Toolkit. 
+
+The Windows Community Toolkit MVVM framework provides a [`RelayCommand`](RelayCommand) type that provides like-for-like functionality taking advantage of the `ICommand` System interface. 
+
+Below are a list of migrations that will need to be performed if being used in your current solution. Where a method or property isn't listed, there is a direct replacement with the same name in the MVVM Toolkit and there is no change required.
+
+The first change here will be swapping using directives in your components.
+
+```csharp
+// MvvmLight
+using GalaSoft.MvvmLight.Command;
+using Galasoft.MvvmLight.CommandWpf;
```

Comment:
If you migrate a WPF application and swap the `using Galasoft.MvvmLight.CommandWpf;` to `using Microsoft.Toolkit.Mvvm.Input;` all your commands using `canExecute` will stop working due to the specific handling of `CommandManager` missing in Windows Community Toolkit MVVM framework!